src/main/java/com/tekenable/config/prime/trial/cancer/glaxo/GlaxoCancerPrimer.java

```diff
@@ -106,7 +106,7 @@ protected void createTrial() {
             add(trialInfo);
         }});
 
-        final TrialSite trialSite1 = new TrialSite(trialInfo,
+        final TrialSite trialSite1 = new TrialSite(trialInfo, 1,
```

Comment:
is it still live request, I put sort order parameter like this because it's logically better place and our primers populating trials are more readable too.